.travis/package_steady_cli.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I do not get where the script is actually called. I expected to find it in `.travis.yaml`, but it does not seem to be there...

Reply:
Yes, I need to add this to .travis.yml. I will do that